src/Symfony/Component/Finder/Glob.php

```diff
@@ -45,7 +45,7 @@ class Glob
      *
      * @return string regex The regexp
      */
-    public static function toRegex($glob, $strictLeadingDot = true, $strictWildcardSlash = true, $delimiter = '#')
+    public static function toRegex($glob, $strictLeadingDot = true, $strictWildcardSlash = true, $delimiter = '#', bool $isCaseSensitive = true)
```

Comment:
Wouldn't it be useful to use a `string $modifier = ''` parameter here, that would allow passing further modifiers to this class?

Reply:
I wouldn't do that. I would rather add more boolean parameters for other modifiers in case they are needed in the future.

Reply:
I'd name this arg $caseSensitive